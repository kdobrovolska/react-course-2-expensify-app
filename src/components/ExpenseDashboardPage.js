import React from 'react';
import ExpenseList from './ExpenseList.js'
import ExpenseListFilters from './ExpenseListFilters.js'
import ExpensesSummary from './ExpensesSummary'

const ExpenseDashboardPage = () => (
  <div>
    <ExpenseListFilters />
    <ExpenseList type='new'/>
    <ExpensesSummary />
  </div>
);
export default ExpenseDashboardPage;