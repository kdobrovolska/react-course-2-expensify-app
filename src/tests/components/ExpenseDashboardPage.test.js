import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage.js';

test('should render dashboard', ()=>{
    const wrapper=shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
});