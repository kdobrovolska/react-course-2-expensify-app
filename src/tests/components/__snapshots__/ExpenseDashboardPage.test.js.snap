// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList)
    type="new"
  />
  <Connect(ExpensesSummary) />
</div>
`;
